components/image/picker/src/index.scss

```diff
@@ -0,0 +1,63 @@
+@import '~@schibstedspain/sui-theme/lib/index';
+
+.sui-ImagePicker {
+
+  &-headImage {
+    background-color: color-variation($c-black, 4);
```

Comment:
Can't you use any of the greys ?
https://github.com/SUI-Components/sui-theme/blob/master/src/settings/_color.scss#L29

Reply:
Sure, and it's open to any modification. Colors are not important at this stage.